src/main/java/com/aws/greengrass/FleetProvisioningByClaimPlugin.java

```diff
@@ -247,4 +269,19 @@ private void writeCertificateAndKeyToPath(CreateKeysAndCertificateResponse respo
             throw new RuntimeException(e);
         }
     }
+
+    private void writeCertificateToPath(CreateCertificateFromCsrResponse response, String rootPath) {
+        try {
+            Path certPath = Paths.get(rootPath, DEVICE_CERTIFICATE_PATH_RELATIVE_TO_ROOT);
+            if (Files.notExists(certPath)) {
+                Files.createFile(certPath);
+            }
+            Files.write(certPath, response.certificatePem.getBytes(StandardCharsets.UTF_8));
+            Platform.getInstance().setPermissions(FileSystemPermission.builder().ownerRead(true).build(), certPath);
+
+        } catch (IOException e) {
+            logger.atError().log("Caught exception while writing certificate to file");
```

Comment:
please include the exception in the log by adding `, e` to the end of the log (pass it as the second parameter to the log call)

Reply:
Done